src/System.IdentityModel.Tokens.Jwt/ClaimsAccessorsDoc.md

```diff
@@ -0,0 +1,85 @@
+﻿
+# Claims
+
+Type | Accessor | Used for authorization | Privacy | Claims
+--   | --       | --                     | --      | --     
+string | GetActor | False |  | JwtRegisteredClaimNames.Actort<BR/> ClaimTypes.Actor
+string | GetBirthdate | False | EUII | JwtRegisteredClaimNames.Birthdate<BR/> ClaimTypes.DateOfBirth
+string | GetEmail | False | EUII | JwtRegisteredClaimNames.Email<BR/> ClaimTypes.Email
+string | GetFamilyName | False | EUII | JwtRegisteredClaimNames.FamilyName<BR/> ClaimTypes.Surname
+string | GetGender | False | EUII | JwtRegisteredClaimNames.Gender<BR/> ClaimTypes.Gender
+string | GetGivenName | False | EUII | JwtRegisteredClaimNames.GivenName<BR/> ClaimTypes.GivenName
+string | GetNameid | False |  | JwtRegisteredClaimNames.NameId<BR/> ClaimTypes.NameIdentifier
+string | GetSub | False |  | JwtRegisteredClaimNames.Sub<BR/> ClaimTypes.NameIdentifier
+string | GetWebsite | False |  | JwtRegisteredClaimNames.Website<BR/> ClaimTypes.Webpage
+string | GetUniqueName | False |  | JwtRegisteredClaimNames.UniqueName<BR/> ClaimTypes.Name
+string | GetObjectId | True | EUPI | "oid"<BR/> "http://schemas.microsoft.com/identity/claims/objectidentifier"
+IEnumerable<string> | GetScopes | True |  | "scp"<BR/> "http://schemas.microsoft.com/identity/claims/scope"
+string | GetTenantId | True | OII | "tid"<BR/> "http://schemas.microsoft.com/identity/claims/tenantid"
+string | GetAcr | False |  | "acr"<BR/> "http://schemas.microsoft.com/claims/authnclassreference"
```

Comment:
>Acr [](http://example.com/codeflow?start=12&length=3)

there is also acrs, and these two acr and acrs cause enough confusion. So my suggestion would be to use full name if possible , like GetAuthncClassReference